x/denommetadata/module.go

```diff
@@ -56,7 +56,7 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONCodec, config client.TxEncod
 	if err := cdc.UnmarshalJSON(bz, &genState); err != nil {
 		return fmt.Errorf("failed to unmarshal %s genesis state: %w", types.ModuleName, err)
 	}
-	return ValidateGenesis(genState)
+	return nil
```

Comment:
Why you return `nil` here? Is there any problem with validate genesis?

Reply:
I've already delet ValidateGenesis because Genesis State is empty now